packages/mdc-switch/mdc-switch.scss

```diff
@@ -45,79 +38,86 @@
       transform: translateX($mdc-switch-knob-active-margin);
     }
   }
+}
+
+.mdc-switch__background {
+  display: block;
+  position: relative;
+  width: $mdc-switch-track-width;
+  height: $mdc-switch-track-height;
+  border-radius: 7px;
+  outline: none;
+  user-select: none;
 
-  &__background {
+  // Track
+  &::before {
     display: block;
-    position: relative;
-    width: $mdc-switch-track-width;
-    height: $mdc-switch-track-height;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    transition:
+      mdc-switch-transition(opacity),
+      mdc-switch-transition(background-color);
     border-radius: 7px;
-    outline: none;
-    user-select: none;
+    opacity: .38;
+    content: "";
+  }
+}
 
-    // Track
-    &::before {
-      display: block;
-      position: absolute;
-      top: 0;
-      right: 0;
-      bottom: 0;
-      left: 0;
-      transition:
-        mdc-switch-transition(opacity),
-        mdc-switch-transition(background-color);
-      border-radius: 7px;
-      opacity: .38;
-      content: "";
-    }
+// postcss-bem-linter: ignore
+.mdc-switch__knob {
+  @include mdc-elevation(2);
+
+  display: block;
+  position: absolute;
+  top: $mdc-switch-knob-vertical-offset_;
+  left: 0;
+  width: $mdc-switch-knob-diameter;
+  height: $mdc-switch-knob-diameter;
+  transform: translateX(0);
+  transition:
+    mdc-switch-transition(transform),
+    mdc-switch-transition(background-color);
+  border-radius: $mdc-switch-knob-diameter / 2;
+  z-index: 1;
+
+  // Focus indicator
+  // postcss-bem-linter: ignore
+  &::before {
+    @include mdc-switch-tap-target_;
 
-    // postcss-bem-linter: ignore
-    .mdc-switch__knob {
-      @include mdc-elevation(2);
-
-      display: block;
-      position: absolute;
-      top: $mdc-switch-knob-vertical-offset_;
-      left: 0;
-      width: $mdc-switch-knob-diameter;
-      height: $mdc-switch-knob-diameter;
-      transform: translateX(0);
-      transition:
-        mdc-switch-transition(transform),
-        mdc-switch-transition(background-color);
-      border-radius: $mdc-switch-knob-diameter / 2;
-      z-index: 1;
-
-      // Focus indicator
-      // postcss-bem-linter: ignore
-      &::before {
-        @include mdc-switch-tap-target_;
-
-        transform: scale(0);
-        transition:
-          mdc-switch-transition(transform),
-          mdc-switch-transition(background-color);
-        border-radius: $mdc-switch-focus-ring-diameter / 2;
-        opacity: .2;
-        content: "";
-      }
-    }
+    transform: scale(0);
+    transition:
+      mdc-switch-transition(transform),
+      mdc-switch-transition(background-color);
+    border-radius: $mdc-switch-focus-ring-diameter / 2;
+    opacity: .2;
+    content: "";
   }
 }
 
+@at-root {
```

Comment:
Let's move this block immediately after the `.mdc-switch` rule above.

(We usually try to keep all the theme-related stuff as close to the top of the file as possible, to make it easier for readers to figure out how theming works.)